charts/prometheus-node-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.0.1
+appVersion: 1.1.2
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.15.0
+version: 1.15.1
```

Comment:
Imho this should be a minor version bump as it's not a Bugfix and the image also had a minor bump.

Reply:
Thanks, changed it